docs/version0.25.md

```diff
@@ -95,6 +96,11 @@ OpenJ9 on AIX&reg; systems uses the hardware-accelerated `zlibNX` if the Nest ac
 
 The `%sysname` dump token is added on z/OS, which equates to the SYSNAME sysparm. See [Dump agent tokens](xdump.md#dump-agent-tokens).
 
+### Single build for compressed references and no compressed references
+
+A single build now supports compressed references and no compressed references (mixedrefs). The mode is selected at runtime based on the specified heap size ([`-Xmx`](#xms.md)), or via pre-existing command line options that control compressed references behavior. A command line option must be used on a mixedrefs build when no compressed references is desired for smaller heap sizes that otherwise select compressed references automatically. This is a change in behavior from using a large heap build. The `compressedrefs` directory is not present in a mixedrefs build.
```

Comment:
Suggestion to move `(mixedrefs)` to earlier in the sentence and add the word `both`:
> A single (mixedrefs) build now supports both compressed references and no compressed references.

Also a suggestion to add `object reference` to the second sentence:
> The object reference mode is selected at runtime...

Reply:
Updated with a couple of other tweaks.